k9mail/src/main/java/com/fsck/k9/provider/UnreadWidgetProvider.java

```diff
@@ -115,7 +124,11 @@ public static void updateWidget(Context context, AppWidgetManager appWidgetManag
             remoteViews.setTextViewText(R.id.unread_count, displayCount);
         }
 
-        remoteViews.setTextViewText(R.id.account_name, accountName);
+        if (folderName != null) {
+            remoteViews.setTextViewText(R.id.account_name, accountName + "/" + folderName);
```

Comment:
1. This should use a format string: getString(R.string.unread_widget_fmt, accountName, folderName);
2. `R.id.accout_name` is becoming an increasingly stupid name for this property. I think it should probably just be title. It can be:
* The app name
* An account name
* The account name followed by the folder name.

If you just call it title you can move this if-clause back into the folder handling stuff above.